Public/Src/Tools/Tool.MsBuildGraphBuilder/MsBuildAssemblyLoader.cs

```diff
@@ -31,48 +32,56 @@ public sealed class MsBuildAssemblyLoader : IMsBuildAssemblyLoader
         private const string SystemCollectionsImmutable = "System.Collections.Immutable.dll";
         private const string SystemThreadingDataflow = "System.Threading.Tasks.Dataflow.dll";
 
-        // MsBuild.exe is not a required to be loaded but is required to be found
-        private const string MsBuildExe = "MsBuild.exe";
+        // MsBuild is not a required to be loaded but is required to be found
+        // Under DotNetCore there is not an msbuild.exe but a msbuild.dll
+        private string m_msbuild;

```

Comment:
This is the value that eventually gets passed to the `PipGraphConstructor` constructor right? Couldn't easily find the callsite in the review.

Reply:
Yes, the location of msbuild ends up going into the graph result, which is later serialized and picked up by the graph constructor

---
In reply to: [301352822](https://github.com/microsoft/BuildXL/pull/561#discussion_r301352822) [](ancestors = 301352822)